### DISCUSSION TITLE
Edit application is set to disallow violence and Sexual acts, but I can still get a model that allows it using the API.

### DISCUSSION THREAD

**Discussion by DavidWeng916** [pixiv/vroid-sdk-developers#99] [Q&A] [answered]
Hello, I would like to ask, my edit application is set to disallow sexual depictions and violent depictions, but when I get the list of models through the `DefaultApi.GetHeartsAsync()` API, I still get that some models have licenses that allow Violence and Sexual acts. Is there something I'm not setting up right?

Like this [model](https://hub.vroid.com/en/characters/7751625412407104345/models/163078324951161659)
It's violence is allowed, my setting is not allowed, but I can still get this model using the `DefaultApi.GetHeartsAsync()` API.

This is my Edit application setting.
<img width="1220" alt="截圖 2023-07-13 下午4 24 15" src="https://github.com/pixiv/vroid-sdk-developers/assets/115971193/0a132372-a263-4b1d-8da7-0c186848b60f">

This is the model's license.
<img width="388" alt="截圖 2023-07-13 下午4 14 14" src="https://github.com/pixiv/vroid-sdk-developers/assets/115971193/f24d6d25-20f6-4122-acc2-652da8ef5895">

**Comment by nyamadan** [comment, marked as answer]
I'm sorry for the late reply.

"Violence : NO" In the VRM permissions means that do not use for applications with  "Usage in violent depictions is YES"

Your application's "Usage in violent depictions" is "NO".
This means that your application can load both "Violence : YES" and "Violence : NO" models.

The same is true for "Usage in sexual descriptions."

**Reply by DavidWeng916** [reply]
I see, I misunderstood what it meant. Thank you for your answer.
